modules/admin_manual/pages/installation/system_requirements.adoc

```diff
@@ -7,134 +7,115 @@
 == Officially Recommended Environment
 
 For _best performance_, _stability_, _support_, and _full functionality_
-we officially recommend:
+we officially recommend and support:
 
 [cols="1,2a",options="header"]
 |===
-| Platform
-| Options
+|Platform
+|Options
 
-| Operating System
-|
-Ubuntu 18.04 LTS
+|Operating System
+|Ubuntu 18.04 LTS
 
-| Database
-|
-MariaDB 10+
+|Database
+|MariaDB 10+
 
-| Web server
-| Apache 2.4 with xref:installation/manual_installation.adoc#multi-processing-module-mpm[`prefork and mod_php`]
+|Web server
+|Apache 2.4 with xref:installation/manual_installation.adoc#multi-processing-module-mpm[`prefork and mod_php`]
 
-| PHP Runtime
-| {recommended-php-version}
+|PHP Runtime
+|{recommended-php-version}
 |===
 
-
 == Officially Supported Environments
 
-For _best performance_, _stability_, _support_, and _full functionality_
-we officially support:
+For _best performance_, _stability_, _support_, and _full functionality_ we officially support:
 
 === Server
 
 [cols="1,2a",options="header"]
 |===
-| Platform
-| Options
+|Platform
+|Options
 
-| Operating System
+|Operating System
 |
-* Ubuntu 16.04 and 18.04
 * Debian 8 and 9
-* SUSE Linux Enterprise Server 12 with SP4 and 15
-* Red Hat Enterprise Linux/Centos 7.5 and 8
 * Fedora 28 and 29
+* Red Hat Enterprise Linux/Centos 7.5 and 8 (*64-bit only*)
+* SUSE Linux Enterprise Server 12 with SP4 and 15
+* Ubuntu 16.04 and 18.04
 * openSUSE Leap 42.3 and 15
 
-| Database
+|Database
 |
-* MySQL 8+ or MariaDB 10+ (RECOMMENDED)
-* Oracle 12g
-* PostgreSQL 9 (versions 10 and above are not _yet_ supported)
-* SQLite (NOT FOR PRODUCTION)
+* MySQL 8+ or MariaDB 10+ (*Recommended*)
+* Oracle 11 and 12
```

Comment:
This seems to be a real change. It already says Oracle 11 and 12 in 10.4 and 10.3 branches, so this is just making it consistent.